src/main/java/creativei/enums/FollowUpSubStatus.java

```diff
@@ -1,5 +1,8 @@
 package creativei.enums;
 
+import java.util.List;
+import java.util.stream.*;
+
 public enum FollowUpSubStatus {
     MORE_COUNSELLING("Need More Cunselling"),
     ENROLLMENT_IN_THIRTY_DAYS("Possible Enrollment In A Month"),
```

Comment:
no need for capitalized words- Possible enrollment in a month

Reply:
should i change all of them or should i change only Possible enrollment in a month

Reply:
issue resolved in commit 680b8b1